sysdump/sysdump.go

```diff
@@ -270,3 +274,22 @@ func IsDirEmpty(name string) (bool, error) {
 
 	return true, nil
 }
+
+// Check the status of KubeArmor
+func CheckStatus(c *k8s.Client) (bool, error) {
+	var status bool
+	pods, err := c.K8sClientset.CoreV1().Pods("kube-system").List(context.Background(), metav1.ListOptions{
```

Comment:
```suggestion
	pods, err := c.K8sClientset.CoreV1().Pods("").List(context.Background(), metav1.ListOptions{
```

Let's check in all namespaces.